lib/src/mt_ptp.c

```diff
@@ -490,6 +490,23 @@ static void ptp_adjust_delta(struct mt_ptp_impl* ptp, int64_t delta, bool error_
   ptp->stat_delta_sum += labs(delta);
 }
 
+static void ptp_delay_req_read_tx_time_handler(void* param) {
```

Comment:
After TSN pacing enabled, IEEE Qbv traffic shaper also enabled for isolating PTP and Video traffic. The shaper will block PTP message transmission till the blank time slice of the end of video frame. So, HW TX timestamp fails to be retrieved immediatly while the PTP message is blocked. It will cause TX timestamping failed error.

    Using alarm to poll TX timestamp can solve the issue.